lib/pure/browsers.nim

```diff
@@ -19,6 +19,11 @@ when defined(windows):
 else:
   import os, osproc
 
+const osOpenCmd* =
+  when defined(macos): "open" elif defined(windows): "start" else: "xdg-open" ## \
```

Comment:
windows still doesn't match the command it actually uses?

Reply:
Fixed.